test/value.cpp

```diff
@@ -2108,6 +2140,7 @@ class value_test
         testStdConstruction();
         testInitList();
         testEquality();
+        testHash();
```

Comment:
array.cpp and object.cpp each need their own small `testHash()` to exercise the corresponding hash function.

Reply:
Done.